deploy/olm-catalog/ibm-management-ingress-operator/1.4.5/ibm-management-ingress-operator.v1.4.5.clusterserviceversion.yaml

```diff
@@ -95,6 +95,11 @@ metadata:
   name: ibm-management-ingress-operator.v1.4.5
   namespace: placeholder
 spec:
+  relatedImages:
+    - name: ICP_MANAGEMENT_INGRESS_IMAGE
+      image: quay.io/opencloudio/icp-management-ingress:2.5.14
```

Comment:
should we update the image version to 2.5.19 to match RELEASE_VERSION in operand repo?

Reply:
I had the same query and checked with Adam from CI/CD. Here is a brief summary from the detailed response he provided "Regardless of what tag is set (or even if a tag is set) as long as the image name and environment variable name are right we will replace whatever is after the image name with the sha." However, it is a good practise to have the latest tags, so will be updating to the info  available in the release pipeline.